prime-dotnet-webapi-tests/Unit Tests/MinorUpdateRules.Tests.cs

```diff
@@ -150,7 +150,12 @@ public async void TestAllowableChangesRule_Certifications()
         [Fact]
         public async void TestAllowableChangesRule_Jobs()
         {
-            Enrollee enrollee = new EnrolleeFactory().Generate();
+            var factory = new EnrolleeFactory();
+            Enrollee enrollee = null;
+            do
+            {
+                enrollee = factory.Generate();
+            } while (enrollee.OboSites.Count < 1);
```

Comment:
We can use a RuleSet to specify no Certifications on an Enrollee rather than randomly hoping to get one that has what we want.

Reply:
Good to know, I changed to use a RuleSet